packages/components/src/components/common/toggleGroup/toggleGroup.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import * as ToggleGroupPrimitive from '@radix-ui/react-toggle-group';
+
+import { cn } from '@/utils/class-name';
+import { toggleGroup } from './toggleGroup.css';
+
+type ToggleGroupProps = React.ComponentPropsWithoutRef<typeof ToggleGroupPrimitive.Root> & {
+  gap?: 1 | 2 | 3 | 4 | undefined;
+};
+
+const ToggleGroup = React.forwardRef<React.ElementRef<typeof ToggleGroupPrimitive.Root>, ToggleGroupProps>(
+  ({ disabled, className, gap, ...props }, ref) => {
+    return (
+      <ToggleGroupPrimitive.Root
+        className={cn(toggleGroup({ gap }), className, 'ui-toggle-group')}
+        disabled={disabled}
+        {...props}
+        ref={ref}
+      />
+    );
+  }
+);
+ToggleGroup.displayName = ToggleGroupPrimitive.Root.displayName;
+
+const ToggleGroupItem = React.forwardRef<
+  React.ElementRef<typeof ToggleGroupPrimitive.Item>,
+  React.ComponentPropsWithoutRef<typeof ToggleGroupPrimitive.Item>
+>(({ className, ...props }, ref) => {
+  return (
+    <ToggleGroupPrimitive.Item ref={ref} className={cn(className, 'ui-toggle-group-item')} {...props}>
+      {props.children}
+    </ToggleGroupPrimitive.Item>
+  );
+});
+ToggleGroupItem.displayName = ToggleGroupPrimitive.Item.displayName;
```

Comment:
the children is already part of props, so just don't do something with it ;)
and same here directly return and save one indent:
```tsx
const ToggleGroupItem = React.forwardRef<
  React.ElementRef<typeof ToggleGroupPrimitive.Item>,
  React.ComponentPropsWithoutRef<typeof ToggleGroupPrimitive.Item>
>(({ className, ...props }, ref) => <ToggleGroupPrimitive.Item ref={ref} className={cn(className, 'ui-toggle-group-item')} {...props} />);
ToggleGroupItem.displayName = ToggleGroupPrimitive.Item.displayName;
```